sample/build.gradle

```diff
@@ -0,0 +1,41 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+//TODO apply spoon plugin
```

Comment:
?

Reply:
TODOčka vyuzivam k tomu aby lidi co si budou cist sample věděli, co vsechno musi udelat - muzes si zobrazit seznam TODO, okoukal sem to z Udacity

Reply:
Tady se už ale nemusí nic dodělat. Ten spoon plugin je aplikovaný o řádek níž

Reply:
Jasně že ne, jde o to, že si zájemce zobrazí seznam todos a uvidí co přesně kde a jak musí udělat ve svém projektu - to napíšu do readme.

Reply:
Ale jestli se vám to nelíbí, klidně to vyhodím. :)

Reply:
Udelej radsi dobry README.md kde popises co a jak udelat